.gitignore

```diff
@@ -2,3 +2,5 @@
 *.swp
 *egg-info
 dist
+
+.vscode
```

Comment:
I'm not a fan of adding this to the gitigore as it's IDE-specific. I tend to add that to my global gitignore.

Reply:
But everyone uses VS Code now! You're right though, I'll rebase it out.